README.md

```diff
@@ -1,2 +1,35 @@
-# proposal
-M3DB Project Design Documents
+# Proposing Changes to M3DB
+
+## Introduction
+
+The design process for changes to M3DB is loosely modeled on the [proposal process used by the Go project](https://github.com/golang/proposal).
```

Comment:
The Go proposal indicates that proposals are needed for "significant changes". Should we mention this well? Is the interpretation of "significant changes" mostly based on the proposer's judgement?

Reply:
Yeah we can mention this.  I think keeping it open to interpretation is probably ideal.  It's going to be quite difficult to quantify precisely up front what that is.